kong/core/balancer.lua

```diff
@@ -27,6 +28,7 @@ local _load_targets_into_memory
 
 -- table holding our balancer objects, indexed by upstream name
 local balancers = {}
+local balancers_initialized = nil
```

Comment:
It would be sweet to declare this in a `do end` block wrapping the below `init` function?